tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala

```diff
@@ -345,7 +398,10 @@ class HoverTermSuite extends BaseHoverSuite {
        |""".stripMargin,
     automaticPackage = false,
     compat = Map(
-      "3" -> "object Foo: app.Outer".hover
+      "3" ->
+        """|Foo
```

Comment:
Looks like we are getting unnecessary expression type here.

Reply:
fixed https://github.com/scalameta/metals/pull/3854/commits/d5d5230b9eef757682ade1138502097def35333d